sklearn/metrics/pairwise.py

```diff
@@ -1826,7 +1826,11 @@ def _pairwise_callable(X, Y, metric, force_all_finite=True, **kwds):
         out = np.zeros((X.shape[0], Y.shape[0]), dtype="float")
         iterator = itertools.combinations(range(X.shape[0]), 2)
         for i, j in iterator:
-            out[i, j] = metric(X[i], Y[j], **kwds)
+            # scipy not yet implemented 1D sparse slices; can be removed and use simply
+            # `arr[ind]` once implemented
```

Comment:
```suggestion
            # scipy has not yet implemented 1D sparse slices; once implemented this can
            # be removed and `arr[ind]` can be simply used.
```